sources/accessible-name-and-description.ts

```diff
@@ -348,18 +351,37 @@ export function computeTextAlternative(
 		// window.getComputedStyle(elementFromAnotherWindow) or if I don't bind it
 		// the type declarations don't require a `this`
 		// eslint-disable-next-line no-restricted-properties
-		getComputedStyle = window.getComputedStyle.bind(window),
+		getComputedStyle: _getComputedStyle = window.getComputedStyle.bind(window),
 		hidden = false,
 	} = options;
+	const cachingGetComputedStyle: GetComputedStyle = (
+		el,
+		pseudoElement,
+	): CSSStyleDeclaration => {
+		// we don't cache the pseudoElement styles
+		if (pseudoElement !== undefined) {
+			return _getComputedStyle(el, pseudoElement);
+		}
```

Comment:
Let's just throw here and adjust callsites to use `uncachedGetComputedStyle` instead. This makes it clearer where we actually call an uncached `getComputedStyle`.